.github/workflows/main.yml

```diff
@@ -50,6 +46,21 @@ jobs:
       - name: remove .npmrc
         if: steps.is_release_commit.outputs.result == 1
         run: rm -rf .npmrc
+        
+      - name: push module tags to github
+        if: steps.is_release_commit.outputs.result == 1
+        run: git push https://${{ secrets.GH_TOKEN }}@github.com/$GITHUB_REPOSITORY.git --follow-tags
```

Comment:
This git push command pushes all tags on current workspace and branch, use tag push command only as it can do something like below:

```bash
Run git push https://***@github.com/$GITHUB_REPOSITORY.git --follow-tags
To https://github.com/satyamyadav/github-releases.git
 * [new branch]      preparing-module-for-npm-publish -> preparing-module-for-npm-publish
 * [new tag]         v1.1.0-6 -> v1.1.0-6
```
which pushes a new branch with the tag.